src/convertToObject.js

```diff
@@ -13,6 +13,27 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const arr = sourceString.trim().slice(0, -1).split(';');
```

Comment:
```suggestion
  const arr = sourceString.split(';');
```
Так тоже должно работать